helpers/prometheus.py

```diff
@@ -132,51 +137,55 @@ def __build_sentry_data_from_api(self):
-                )
-                log.debug(
-                    "metadata: getting issues from api - project: {proj} env: {env} age: 14d".format(
-                        proj=project.get("slug"), env=env
-                    )
-                )
-                project_issues_14d = self.__sentry_api.issues(
-                    self.org.get("slug"), project, env, age="14d"
-                )
-
-                log.debug("data structure: building projects issues data")
-                for k, v in project_issues_1h.items():
-                    projects_issue_data[project.get("slug")][k] = {"1h": v}
-
-                for k, v in project_issues_24h.items():
-                    projects_issue_data[project.get("slug")][k].update({"24h": v})
-
-                for k, v in project_issues_14d.items():
-                    projects_issue_data[project.get("slug")][k].update({"14d": v})
-
-        data["projects_data"] = projects_issue_data
+        if self.issue_metrics == "True":
+            __metadata = data.get("metadata")
+
+            projects_issue_data = {}
+
+            for project in __metadata.get("projects"):
+                projects_issue_data[project.get("slug")] = {}
+                envs = __metadata.get("projects_envs").get(project.get("slug"))
+                for env in envs:
+                    project_issues_1h = project_issues_24h = project_issues_14d = {}
+                    if self.get_1h_metrics == "True":
+                        log.debug(
+                            "metadata: getting issues from api - project: {proj} env: {env} age: 1h".format(
+                                proj=project.get("slug"), env=env
+                            )
+                        )
+                        project_issues_1h = self.__sentry_api.issues(
+                            self.org.get("slug"), project, env, age="1h"
+                        )
+                    if self.get_24h_metrics == "True":
+                        log.debug(
+                            "metadata: getting issues from api - project: {proj} env: {env} age: 24h".format(
+                                proj=project.get("slug"), env=env
+                            )
+                        )
+                        project_issues_24h = self.__sentry_api.issues(
+                            self.org.get("slug"), project, env, age="24h"
+                        )
+                    if self.get_14d_metrics == "True":
+                        log.debug(
+                            "metadata: getting issues from api - project: {proj} env: {env} age: 14d".format(
+                                proj=project.get("slug"), env=env
+                            )
+                        )
+                        project_issues_14d = self.__sentry_api.issues(
+                            self.org.get("slug"), project, env, age="14d"
+                        )
+
+                    log.debug("data structure: building projects issues data"):
```

Comment:
Just noticed that I've accidentally left a colon here so the code probably won't run 🤦🏽 @italux Instead of making a new PR it's probably easier for you to just amend it whenever you get a chance. Sorry about that!